test/jdk/java/awt/List/MouseDraggedOriginatedByScrollBarTest.java

```diff
@@ -77,27 +82,62 @@ private static Frame createTestUI() {
             new MouseMotionAdapter(){
                 @Override
                 public void mouseDragged(MouseEvent me){
-                    PassFailJFrame.log(me.toString());
+                    System.out.println(me.toString());
+                    throw new RuntimeException("Test failed. Mouse dragged " +
+                            "event detected.");
                 }
             });
 
         list.addMouseListener(
             new MouseAdapter() {
                 public void mousePressed(MouseEvent me) {
-                    PassFailJFrame.log(me.toString());
+                    System.out.println(me.toString());
+                    throw new RuntimeException("Test failed. Mouse pressed " +
+                            "event detected.");
                 }
 
                 public void mouseReleased(MouseEvent me) {
-                    PassFailJFrame.log(me.toString());
+                    System.out.println(me.toString());
+                    throw new RuntimeException("Test failed. Mouse released " +
+                            "event detected.");
                 }
 
                 public void mouseClicked(MouseEvent me){
-                    PassFailJFrame.log(me.toString());
+                    System.out.println(me.toString());
+                    throw new RuntimeException("Test failed. Mouse clicked " +
+                            "event detected.");
                 }
             });
 
         frame.setLayout(new FlowLayout());
         frame.pack();
-        return frame;
+        frame.setLocationRelativeTo(null);
+        frame.setVisible(true);
+    }
+
+    private static void test() throws Exception {
+        Robot robot;
+        try {
+            robot = new Robot();
+        } catch (Exception ex) {
+            throw new RuntimeException("Can't create robot");
+        }
```

Comment:
```suggestion
        Robot robot = new Robot();
```
The `test` method has the `throws` clause with `Exception`, just let `AWTException` propagate.